kotlin-runtime/external-module-template/target.tmpl/generated-sources/ftl/{{ range .ExternalModules }}{{ push .Name . }}{{ end }}/{{ .Name | camel }}.kt

```diff
@@ -34,8 +34,19 @@ data class {{.Name|title}}
 {{- else if is "Verb" . }}
 {{.Comments|comment -}}@Verb
 @Ignore
+{{- if and (eq (type $ .Request) "Unit") (eq (type $ .Response) "Unit")}}
+fun {{.Name|lowerCamel}}(context: Context): Unit = throw
```

Comment:
You could probably pull `fun {{.Name|lowerCamel}}(context: Context` out from the if/else bodies if you wanted to DRY this a bit more.

Reply:
I'll noodle on this one a bit. It makes the template quite a bit tougher for me to read, but maybe I'm just weird :)

Reply:
I was wondering that too. Just leave it as-is if it's too gross.